account/iban-integration.mdx

```diff
@@ -36,129 +49,80 @@ Monerium requires a specific message to be signed by the User's wallet.
 curl -X GET /api/v1/ibans/signing-message
 ```
 
-To request signature from your gnosis safe, you can follow this [demo signature implementation](#message-signing-example-with-gnosispay-account-kit).
+To request signature from the User's wallet, you can follow this [demo signature implementation](#message-signing-example-with-viem).
 
 <Warning>
 We cannot request new IBANs nor transfer existing IBANs until the message signature is completed. This signature is a mandatory requirement from Monerium to verify ownership of the address.
 </Warning>
+
 </Step>
-<Step title="Request the IBAN from Monerium">
 
+<Step title="Request the IBAN from Monerium">
 
-[You need to call this endpoint to request your IBAN profile](): 
+[Make a POST request to endpoint to request IBAN integration for Gnosis Pay user](https://docs.gnosispay.com/api-reference/iban/create-a-new-monerium-integration):
 
 ```bash cURL
-curl -X POST /api/v1/ibans/monerium-profile \
--d '{
-  "signature": "... signed message as a string ...",
-  "callbackUrl": "https://your-app.com/iban-callback"
+curl --request POST \
+  --url https://api.gnosispay.com/api/v1/integrations/monerium \
+  --header 'Authorization: Bearer <token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "signature": "<string>"
 }'
 ```
+If the request is successful, you will receive a response with the IBAN details.
+
 <Warning>
-**This endpoint can fail and it is expected:** Monerium only allows the user to have one single account with them. If the user already has an account, this call will fail to indicate you should follow the oAuth flow with Monerium.
+Monerium only allows the user to have one single account with them. If monerium account **already exists**, then there is **no need** to call this endpoint.
```

Comment:
i would also mention here that access can be granted by users using the monerium api